File: src/lib/scala/Slate.Common/src/main/scala/slate/common/lex/TokenType.scala
/**
<slate_header>
  author: Kishore Reddy
  url: https://github.com/kishorereddy/scala-slate
  copyright: 2016 Kishore Reddy
  license: https://github.com/kishorereddy/scala-slate/blob/master/LICENSE.md
  desc: a scala micro-framework
  usage: Please refer to license on github for more info.
</slate_header>
  */

package slate.common.lex

/**
  * Created by kreddy on 3/2/2016.
  */
object TokenType {

  val None         = 0
  val Ident        = 1
  val String       = 2
  val Number       = 3
  val Boolean      = 4
  val NonAlphaNum  = 5
  val NewLine      = 6
  val End          = 7
  val ParamRef     = 8
  val Interpolated = 9
  val Comment      = 10
}

File: src/common/common/src/main/kotlin/kiit/common/lex/TokenType.kt
/**
 *  <kiit_header>
 * url: www.kiit.dev
 * git: www.github.com/slatekit/kiit
 * org: www.codehelix.co
 * author: Kishore Reddy
 * copyright: 2016 CodeHelix Solutions Inc.
 * license: refer to website and/or github 
 *  </kiit_header>
 */

package kiit.common.lex

object TokenType {
    const val None = 0
    const val Ident = 1
    const val String = 2
    const val Number = 3
    const val Boolean = 4
    const val NonAlphaNum = 5
    const val NewLine = 6
    const val End = 7
    const val ParamRef = 8
    const val Interpolated = 9
    const val Comment = 10
}
